src/app/components/did/encrypt-keys/encrypt-keys.component.ts

```diff
@@ -63,15 +75,16 @@ export class EncryptKeysComponent extends BaseComponent implements OnInit {
   }
 
   encryptKeys() {
-    if (this.encryptForm.invalid) {
-      return;
-    }
-
-    this.keysService
-      .encryptKeys(this.password.value)
-      .subscribe(encryptedFile => {
-        this.encryptedFile = this.postProcessDidFile(encryptedFile);
-        this.encryptForm.reset();
+    this.vaultService.saveDIDToVault(
+      this.didService.getId(),
+      this.managementKeys,
+      this.didKeys,
+      this.services,
+      '123qweASD!@#').subscribe((result: ResultModel) => {
```

Comment:
Is this a leftover from debugging? We should use `this.password.value`, shouldn't we?

Reply:
It is left over for testing purposes. The encrypt-keys component will be deprecated.